.circleci/config.yml

```diff
@@ -46,22 +54,6 @@ executors:
       NODE_ENV: development # Needed if playwright is in devDependencies
 
 commands:
-  app-build-and-store:
```

Comment:
This `app-build-and-store` command is used only by `app-build-and-store-job` job. To make things lesser confusing, I'm removing this command and put all steps inside the `app-build-and-store-job` job.